### PR TITLE
[BBC-9093] SuccessScreen.MDX

### PR DESCRIPTION
**Pull request by wakooka**
## Description

Migrated successScreen to mdx

Kapture 2020-11-02 at 16 04 01

## Things to consider

Removed `elements` and use successModal.style instead.

## How it was tested

Locally in storybook

### CODE REVIEW

**Comment by gabrielagabriel** [comment]
Hey! Can we have an exemple of the screen opened?

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "7.7.5",
+    "@babel/cli": "7.12.1",
```
Comment (mcellier) [inline package.json:45]: unrelated change

File: src/successModal/SuccessModal.story.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { BaseSection } from '../layout/section/baseSection'
+import { SuccessModal } from './index'
+
+<Meta title="Widgets|SuccessModal" />
```
Comment (mcellier) [inline src/successModal/SuccessModal.story.mdx:7]: nit: regroup all modals to a common group

**Review by mcellier — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mcellier) [reply package.json:45]: - [ ] Change

Comment (wakooka) [reply package.json:45]: 🤔 Thanks I'll check why

**Comment by wakooka** [comment]
> Hey! Can we have an exemple of the screen opened?

An example in the story?

File: src/successModal/SuccessModal.story.mdx (hunk 2 above)
Comment (gabrielagabriel) [inline src/successModal/SuccessModal.story.mdx:7]:
```suggestion
<Meta title="Screens|SuccessModal" />
```

I've created a category Screens for all those screens (even though we call them modals)

File: src/successModal/SuccessModal.story.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### Layout
+
+This modal is always being displayed in fullscren. Use `size` prop to change the layout depending on the screen size.
+
+## Usage
+
+```jsx
+import { SuccessModal } from '@blablacar/ui-library/build/successModal'
+;<SuccessModal />
```
Comment (gabrielagabriel) [inline src/successModal/SuccessModal.story.mdx:68]: mind the `; `

**Comment by wakooka** [comment]
Why the renamed @gabrielagabriel https://github.com/blablacar/ui-library/pull/940/files#diff-c9b324b6d136f5246cb3789903c03c7664d109ccff2ddba9e698ce3be75e76aa ?

File: src/successModal/SuccessModal.style.tsx [hunk 4]
```diff
@@ -34,7 +34,7 @@ const StyledSuccessModal = styled(Modal)`
 `
 
 /* LAYOUT: Media */
-const Media = styled.div`
+export const Media = styled.div`
```
Comment (gabrielagabriel) [inline src/successModal/SuccessModal.style.tsx:37]: IMHO we should export an **StyledSuccessModal** object to avoid having all components as pure components since they are part of **SuccessModal**

**Review by gabrielagabriel — CHANGES_REQUESTED** [review]
Review styled-components exports

File: src/successModal/SuccessModal.style.tsx (hunk 4 above)
Comment (wakooka) [reply src/successModal/SuccessModal.style.tsx:37]: How can I do that?

Comment (wakooka) [reply src/successModal/SuccessModal.style.tsx:37]: Ah! or do you mean to keep it as before with the 
```
export const SuccessModalElements = {
  StyledSuccessModal,
  Media,
  Figure,
  Content,
  SuccessTitle,
  SuccessAction,
  SuccessButton,
}
```

**Comment by wakooka** [comment]
> Review styled-components exports

Done

**Comment by gabrielagabriel** [comment]
@wakooka  I'm renaming to follow commit conventions (cf. Slack)
